recurrence/base.py

```diff
@@ -842,7 +842,7 @@ def serialize_rule(rule):
         if rule.interval != 1:
             values.append((u'INTERVAL', [str(int(rule.interval))]))
         if rule.wkst:
-            values.append((u'WKST', [Rule.weekdays[rule.wkst]]))
+            values.append((u'WKST', [Rule.weekdays[rule.wkst.weekday]]))
```

Comment:
The failed tests are interesting. I'm running python 3.4 and Django 1.8, and here, `rule.wkst` is a `Weekday` instance, but in some of the test suites, this seems to be an int.

Any ideas what I may be doing wrong?